hack/pre-release.sh

```diff
@@ -303,6 +318,92 @@ dist_git_update() {
   echo
 }
 
+# This function handles updating the versions in gitlab. Expects you to be connected to the VPN
+# takes 2 arguments
+# 1st arg: stream to update. Can be x, y or z to represent the x y or z stream.
+# 2nd arg: current WMCO version
+# 3rd arg: updated WMCO version
+gitlab_update() {
+ if [ "$#" -ne 3 ]; then
+    echo incorrect parameter count for dist_git_update $#
+    return 1
+  fi
+  stream=$1
+  wmco_version=$2
+  updated_version=$3
+  OCP_version=$(get_OCP_version $wmco_version)
+
+  current_dir=$PWD
+
+  # clones the gitlab repo into a temporary directory
+  temp=$(mktemp -d)
+  cd $temp
+  clone_repo "https://gitlab.cee.redhat.com/cpaas-products/openshift-winc.git"
+
+  GITLAB_LOCATION=$temp/openshift-winc
+  cd $GITLAB_LOCATION
+
+  latest_hash=$(git rev-parse HEAD)
+
+  gitlab_initial_branch=$(git branch --show-current)
+  new_branch=$(create_git_branch_list $stream-stream-bump $gitlab_initial_branch)
+  switch_git_branch $gitlab_initial_branch $new_branch
+
+  git mv versions/release-$wmco_version versions/release-$updated_version
+  rm versions/release-$updated_version/advisory_map.yml
+
+  # Example: matches "Windows Containers 8.0.1"
+  sed -i "s/Windows Containers $wmco_version/Windows Containers $updated_version/" versions/release-$updated_version/release.yml
+
+  commit_message="Update version to $updated_version
+
+  This commit was generated by hack/pre-release.sh"
```

Comment:
Does this actually create a well formed commit message? I suspect there will be extra whitespaces added which are not needed.